business_objects/personal_access_token.py

```diff
@@ -0,0 +1,83 @@
+import datetime
+from typing import List
+from ..session import session
+from submodules.model.business_objects import general
+from submodules.model.models import PersonalAccessToken
+from datetime import date
+
+
+def get(project_id: str, user_id: str, name: str) -> PersonalAccessToken:
```

Comment:
usually, our get methods use project_id and table id (in this case token_id)

this one would be something like get_by_user_and_name I think

Reply:
agree, changed it to get_by_user_and_name